ibm-colors.json

```diff
@@ -1,172 +0,0 @@
-{
```

Comment:
Understood. I've reinstated _ibm-colors.scss as before and ibm-colors.json, as literal files NOT generated from the template, for deletion in a future version.